src/utils/spo.spec.ts

```diff
@@ -1955,6 +1964,69 @@ describe('utils/spo', () => {
     assert.strictEqual(id, 'contoso.sharepoint.com,ea49a393-e3e6-4760-a1b2-e96539e15372,66e2861c-96d9-4418-a75c-0ed1bca68b42');
   });
 
+  it('returns the folder server relative URL by URL', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
```

Comment:
Over all the tests that we added, let's try to not reuse the same strings too much and rather initialize a variable.